terraform/cloud-platform-components/prometheus.tf

```diff
@@ -0,0 +1,74 @@
+resource "helm_repository" "coreos" {
+  name = "coreos"
+  url  = "https://s3-eu-west-1.amazonaws.com/coreos-charts/stable/"
+}
+
+resource "kubernetes_storage_class" "prometheus" {
```

Comment:
Should we have a retain `reclaimPolicy` ?

Reply:
Not sure I can set a `reclaimPolicy` in tf provider:
https://www.terraform.io/docs/providers/kubernetes/r/storage_class.html#storage_provisioner